### PR TITLE
Afsha Sethna: Coding fun completed

### PR DESCRIPTION
**Pull request by asethna**
Hi Tony and Team,

Please see my GIT changes.

Adding to game:
 - README_: defining information of how to play the game and testing
 - game.py: game script to play rock, paper, scissors
 - test_game.py: test script use to test game.py script using pytest
Adding to parser:
 - README_: defining information of how the statistics script works 
   and what it calculates. Providing input and testing information.
 - parser.py: script calculates statistics of streaming rate log files

Should there be any issues or problem, please do let me know.

FYI, this was fun! :)

Kind Regards,
Afsha Sethna
Email: afshasethna@gmail.com

### CODE REVIEW

**Comment by tonywgh** [comment]
Thanks for submitting the pull-request. I'll take a look soon.

File: game/game.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    '''
+    Summary:    Compares the user_input with the computer generate input
+                to determine the winner
+    Input:      User provides a character/string
+    Output:     Following the rules defined:
+                    Rock wins by blunting Scissors
+                    Paper wins by covering Rock
+                    Scissors wins by cutting Paper
+                    Both are same it is a draw
+                the winner will be determined.
```
Comment (tonywgh) [inline game/game.py:22]: very nice docstrings!

File: game/game.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                return "Scissors cut paper\nComputer Wins!"
+            else:
+                return "Paper covers rock\nUser Wins!"
+        elif user_input == 's':
+            if computer_input == 'r':
+                return "Rock blunts scissors\nComputer Wins!"
+            else:
+                return "Scissors cut paper\nUser Wins!"
+        else:
+            return "Invalid user input: %s" % user_input
```
Comment (tonywgh) [inline game/game.py]: It would be better to validate user input before comparing inputs

File: game/test_game.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                when paper is picked.
+    Input:      None
+    Output:     None
+    '''
+    def _mocked():
+        return 1
+    monkeypatch.setattr("game.generate_random", _mocked)
+    assert "Draw!" in game.compare('p')
+    
+def test_draw_scissors(monkeypatch):
```
Comment (tonywgh) [inline game/test_game.py]: Very nice to see you are trying to test all scenarios. But if you make use of pytest fixtures, you actually only need 1 test function for 3 testing draws.

**Comment by asethna** [comment]
Hi @tonywgh,

Thank you for taking time out to do a thorough review and providing great feedback. I appreciate it! Using pytest was a fairly new learning experience for me. Should I make the changes according to your suggestion.

Thank-you,
Afsha

**Comment by tonywgh** [comment]
Code review is part our daily tasks. I've learned something new from your code. If you want to try pytest fixtures to simplify the test code, please go ahead.

**Comment by asethna** [comment]
@tonywgh, pytest fixtures is interesting addition of pytest. I hadn't use it before. It is really powerful. Made the changes in game.py and test_game.py. Have a great day!

**Pull request closed without merging**